script/Ciel/Day 7/07_01_CIEL07_1.txt

```diff
@@ -255,22 +264,24 @@ I involuntarily close my mouth just as I'm about to ask for what I want.
 -- Page 20, Offset 14850.
 -- 「はい、お任せください。いざともなれば秋葉さまの分をちょ～っとだけ少な目にするのもやむなしです！」
 // はい、お任せください。 here might be too liberal for some?
-"Sure, no problem. In the worst case scenario, I might have no other choice but to take %{i}just%{/i} a little from Akiha-sama's portion!"
+"Okay, I won't disappoint.
```

Comment:
as trite as it is, I might go for "leave it to me" instead of "I won't disappoint." 
I'm guessing you were going for something with a more formal register to match

Reply:
I just didn't want to use `leave it to me` because Shiki says `I'll leave it to you right before this

Reply:
good point. it can stay